src/net.cpp

```diff
@@ -42,6 +43,7 @@
 static_assert(MINIUPNPC_API_VERSION >= 10, "miniUPnPc API version >= 10 assumed");
 #endif
 
+#include <algorithm>
```

Comment:
I compile without ?

Reply:
The added code uses functions from `https://en.cppreference.com/w/cpp/header/algorithm`. Per the developer notes, every `.cpp` and `.h` file should `#include` every header file it directly uses classes, functions or other definitions from, even if those headers are already included indirectly through other headers.